db/migrate/20241214075118_create_rag_auto_answerings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRagAutoAnswerings < ActiveRecord::Migration[7.2]
```

Comment:
6e9b6ad39d3d3feed9344520119e4522aa94b8f9 have a single migration file to add in the RagWise auto answering instead of 4 migration files

Reply:
Combined